Documentation/source/guide/tutorials-imagesequence.rst

```diff
@@ -0,0 +1,209 @@
+- Select RGB/RGBA, 8/16 Color depth, and preferred compression NONE/Any.
+
+Full instructions on how to use the Blender VSE can be found here https://docs.blender.org/manual/en/latest/video_editing/index.html
+
+
+Adobe Media Encoder
+~~~~~~~~~~~~~~~~~~~
+- Open media encoder
+- Add source video to the queue
+- Set the output format to OpenEXR.
+- Set compression to "Zip"
+- If the source has an alpha channel be sure to scroll down to the bottom of the Video section of the Export Settings and check "Include Alpha Channel"
+- Close the Export Settings by clicking Ok and press the Start Queue button.
+
+How To Convert Image Sequences To Video Files
+=============================================
+
+FFmpeg
+~~~~~~
+Converting your images to video follows a similar process to doing the reverse.
+
+Open a terminal and navigate to the location containing the images.
+In the terminal type:
+
+``ffmpeg -i input_%05d.png output.mp4``
+
+Change ``input_`` to match the name of the files. The number of characters in the sequence (%05d) should match the amount in your input files. For example, if the files have four characters in their sequence (e.g. input_0001.png) then it should use %04d.
+
+For this to work correctly all of the files need to be sequentially numbered and the sequence should start from either 0 or 1.
+
+It is can also specify the framerate and the codec, here is an example for framerate 30fps:
+
+``ffmpeg -framerate 30 -i input%04d.png -c:v libx264 -r 30 -pix_fmt yuv420p out.mp4``
+
+Using an NLE
+============
+
+Kdenlive
+~~~~~~~~
+
+Shotcut
+~~~~~~~
+
+Da Vinci Resolve
+~~~~~~~~~~~~~~~~
+
+Blender VSE
+~~~~~~~~~~~
+For Blender VSE, (go to render tab>file format FFmpeg Video>your preferred codec and container)
+
+Adobe Premiere
+~~~~~~~~~~~~~~
+Import the image "as sequence" in the timeline and render in your preferred video format. 
+
+
+Creating Digital Intermediate For Editing Servers
+=================================================
+
+For Digital Intermediate, the `PRORES 4444` codec is a nice choice for MOV containers. It supports 12-bit with YUVA and retains alpha with 16-bit precision.
+
+It can be done with ffmpeg or in kdenlive/Shotcut importing the TIFF/PNG as sequence.
+
+
+FFmpeg
+~~~~~~
+``ffmpeg -framerate 30 -i input%03d.tiff -f mov -acodec pcm_s16le -vcodec prores_ks -vprofile 4444 -vendor ap10 -pix_fmt yuva444p10le out.mov``
+
+Shotcut/Kdenlive
+~~~~~~~~~~~~~~~~
+- A render profile needs to be created first with below profile:
+- ``f=mov acodec=pcm_s16le vcodec=prores_ks vprofile=4444 vendor=ap10 pix_fmt=yuva444p10le qscale=%quality``
+- Use TIFF/PNG image as sequence in the timeline.
+- Then Render with this newly created prores 4444 profile.
+
+A Tutorial on PRORES in LINUX by CGVIRUS:
+https://youtu.be/oBiaBYthZSo
+
+It can be done with Adobe Premiere/avid/fcpx/resolve etc as well by importing TIFF/PNG as sequence and render as MOV prores 4444.
+
+DaVinci Resolve, Adobe Premiere etc
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
For Davinci Resolve, please use the Transcode method mentioned in the comment above.